exporter/datadogexporter/metrics_exporter.go

```diff
@@ -66,6 +70,36 @@ func newMetricsExporter(ctx context.Context, params component.ExporterCreateSett
 	return &metricsExporter{params, cfg, ctx, client, tr}
 }
 
+func (exp *metricsExporter) pushSketches(ctx context.Context, sl sketches.SketchSeriesList) error {
+	payload, err := sl.Marshal()
+	if err != nil {
+		return fmt.Errorf("failed to marshal sketches: %w", err)
+	}
+
+	req, err := http.NewRequestWithContext(ctx,
+		http.MethodPost,
+		exp.cfg.Metrics.TCPAddr.Endpoint+sketches.SketchSeriesEndpoint,
+		bytes.NewBuffer(payload),
+	)
+	if err != nil {
+		return fmt.Errorf("failed to build sketches HTTP request: %w", err)
+	}
+
+	utils.SetDDHeaders(req.Header, exp.params.BuildInfo, exp.cfg.API.Key)
+	utils.SetExtraHeaders(req.Header, utils.ProtobufHeaders)
```

Comment:
I don't see any compression step so I assume the protobuf payload is sent uncompressed (which is consistent with the header `"Content-Encoding": "identity"` that's set here). At some point you'll probably want to add compression (`zlib` is used in the datadog-agent). Applies to timeseries payloads as well, if that's not already implemented.

Reply:
It is sent uncompressed. Agreed that we can do this in the future. Added AP-1279 to track this.